### PR TITLE
Feat/product

### PR DESCRIPTION
**Pull request by jeanevabejar**
- Modify Order Page UI
- Add Error Toast to Order Page
- Resolve Conflict with Main 
- Working Image Input

### CODE REVIEW

File: src/App.tsx
Comment (louisedungca) [inline src/App.tsx:1]: no need for leading slash. see react-router docs: https://reactrouter.com/en/main/start/tutorial#the-contact-route-ui

**Review by louisedungca — APPROVED** [review]
FEAT PRODUCTS???! - Maja

File: src/components/Navbar.tsx
Comment (cyrilcero) [inline src/components/Navbar.tsx:1]: bumalik po sa dati yung nav bar?

image

**Review by cyrilcero — APPROVED** [review]
Orayt. Ok na po. 👏

**Pull request merged**